embedded-oracle-xe/src/test/resources/application-disabled.properties

```diff
@@ -0,0 +1,4 @@
+spring.datasource.driver-class-name=oracle.jdbc.driver.OracleDriver
```

Comment:
why we need to have these settings for container which is not supposed to be bootstrapped and what is
```jdbc:oracle:thin://localhost:1521/test``` pointing on?